src/classes/MTCH_FindGifts_CTRL.cls

```diff
@@ -27,13 +27,13 @@
     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
     POSSIBILITY OF SUCH DAMAGE.
 */
-/*
+/**
```

Comment:
Will extend this and the previous one. Apex Doc requires at least two asterisks to pick up the class block (https://github.com/SalesforceFoundation/ApexDoc#documenting-class-files) so with just one, the class-level attributes weren't being picked up and they were going into Misc.